### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by cam4673920** [Homebrew/discussions#4621] [Getting started] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.26
ORIGIN: https://github.com/Homebrew/brew
HEAD: eff45ef570f265e226f14ce91da72d7a6e7d516a
Last commit: 7 days ago
Core tap JSON: 04 Jul 21:23 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/2.6.10_1/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.5.1-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew
```

### Description of issue

I am trying to install brew and it is saying that I need to keep entering the command 'eval "$(/opt/homebrew/bin/brew shellenv)" 'everytime i try to install it and it is not adding brew to my path. So everytime I want to use brew I have to reinstall it. What should i do to fix it?

**Comment by SMillerDev** [comment]
Follow the full instructions and add that to your shell profile.

**Comment by osalbahr** [comment]
> I am trying to install brew and it is saying that I need to keep entering the command 'eval "$(/opt/homebrew/bin/brew shellenv)" 'everytime i try to install it and it is not adding brew to my path.

What exactly did it say? Could you paste the full output? I don't remember the install script saying that you have to keep entering this command. Running the `eval` command is only one part of the instructions. There should be another command right above it that makes the `eval` command run automatically every time you open a new Terminal. You can obtain the command by running the installer (running `brew doctor` might also do that, not sure).

**Reply by cam4673920** [reply]
Next steps:
- Run this command in your terminal to add Homebrew to your PATH:
    eval "$(/opt/homebrew/bin/brew shellenv)"
- Run brew help to get started
- Further documentation:
    https://docs.brew.sh

The other command is gone and only the eval command is there now.

**Reply by osalbahr** [reply]
Oh, I see. Yeah, that's odd, for some reason `brew` thinks you already did that step. What do `echo $0` and `brew doctor` tell you?

**Reply by cam4673920** [reply]
it tells me "Your system is ready to brew."

**Reply by osalbahr** [reply]
> it tells me "Your system is ready to brew."

Ok. What about `echo $0`?

**Reply by cam4673920** [reply]
All it says is bash.

**Reply by osalbahr** [reply, marked as answer]
If it says `bash` (not `-bash`) then you'd want to run the following:
```console
$ (echo; echo 'eval "$(/opt/homebrew/bin/brew shellenv)"') >> ~/.bashrc
```
If it says `-bash` use `.bash_profile` instead of `.bashrc`.

Now you won't need to manually run the `eval` command each time you open your Terminal.

**Reply by cam4673920** [reply]
Omg it worked. Thank you so muchhhhhh!

**Reply by osalbahr** [reply]
You’re welcome!
